app/controllers/katello/concerns/api/v2/content_overrides_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Katello
+  module Concerns
+    module Api::V2::ContentOverridesController
+      extend ActiveSupport::Concern
+
+      # overriding_object => pass it either an activation key or content host
+      def validate_content_overrides_enabled(content_params, overriden_object)
+        valid_values = ["yes", "no", "0", "1", "default"]
```

Comment:
The API does not specify yes and no as valid values. Where does the discrepancy come in?